rapidtide/correlate.py

```diff
@@ -56,13 +47,6 @@
     numbaexists = False
 numbaexists = False
 
-try:
-    import nibabel as nib
-
-    nibabelexists = True
-except ImportError:
-    nibabelexists = False
-
```

Comment:
Unused import flagged by the linter. Also, since `nibabel` is a required dependency, this conditional import could be removed from _all_ files.

Reply:
I've now removed this everywhere it occurs.  What do you think of just requiring the other optional dependencies (numba and pyfftw)?  It's not like they're hard to find.  I'm not sure about memory_profiler - that's just something I have used for debugging, don't know if normal people care about it, although if they use SLURM, it can be helpful.  Maybe I could just require that too and be done with it.

Reply:
That sounds like a good idea. It will definitely simplify things.